### ISSUE TITLE
Build Failing

### ISSUE THREAD

**Issue by benjchristensen** [ReactiveX/RxJRuby#2]
The build is currently failing

```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/benjchristensen/development/github/ReactiveX/RxJRuby/build.gradle' line: 13

* What went wrong:
A problem occurred evaluating root project 'rxjruby'.
> Could not find method rspec() for arguments [org.jruby:jruby-complete:1.7.4] on root project 'rxjruby'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

**Comment by benjchristensen** [comment]
@quidryan This was working in the previous build so I'm obviously just doing something wrong in applying the JRuby `rspec` settings. How do I do this with Nebula?

Previous: https://github.com/ReactiveX/RxJava/blob/0.20.x/language-adaptors/rxjava-jruby/build.gradle

**Comment by quidryan** [comment]
Getting the rspec configuration wasn't too hard, you still need to create a task to run the rspecs. I've seen a few plugins out there support depending on gems with some examples of running rspecs. I don't know much about them though.

**Comment by avsej** [comment]
I think this ticket can be fixed as well as pull request opened now.

**Comment by avsej** [comment]
Fixed by #4
